### ISSUE TITLE
Date parameters in reports resets to wrong format/locale

### ISSUE THREAD

**Issue by stuza** [salesagility/SuiteCRM#1786]
My current system locale is set to UK. The Administrator account (that I am using to create the reports) is also set to a UK locale. 

I have created a report for a custom module called "Donations". One of the fields on a Donation is "donation_date" and I have created a report where donation_date is greater than or equal to a given value and less than or equal to another given value. When I hit "update", the report does not display correctly and is different between browsers. In Firefox, Seamonkey and Safari, the "value" boxes become empty and no Donations are displayed. In Chrome, the top "value" box resets to US date format, the bottom one goes blank, and all donations are displayed regardless of date. If instead of using the date selector, I type a date in the format y-m-d then the report works correctly in all browsers. However it will reset the date to d-m-y and pressing update again will cause it to fail.

To recap, what I would expect to see is the ability to use the date selector in the normal fashion but instead it takes my selection, puts it in the wrong format and the report then does not respect those dates anyway.

Steps to Reproduce
1. Set system and administrator as UK locale
1. Create a custom module with a custom date field
2. Create a report where this date field must be >= a given date AND <= another given date.
3. Run the report

My Scenario:
I am trying to create a report for a charity so they can see all donations between two given dates.

Environment:
* SuiteCRM Version used: 7.6.4
* Browser name and version: All browsers, subtly different behavior
* Environment name and version: MySQL, PHP 5.6
* Operating System and version: Debian 8

**Comment by gunnicom** [comment]
Maybe duplicate of https://github.com/salesagility/SuiteCRM/issues/1624

**Comment by stuza** [comment]
It looks to be related but I don't think it's strictly speaking a duplicate.

**Comment by vladbar** [comment]
Hi stuza, I've read the description of this issue and it looks like it is a duplicate of case #1624.

**Comment by ijdavie** [comment]
This appears to be a duplicate of #1624, I will close this case, you would like to add anything to issue 1624, please do so. There is also a fix for issue 1624.
